From 5c02345f465cf8da56ffb8d352cebcf7201999ca Mon Sep 17 00:00:00 2001
From: Amit Kumar Mahapatra <amit.kumar-mahapatra@xilinx.com>
Date: Wed, 16 Sep 2020 14:24:51 -0600
Subject: [PATCH 1595/1851] mtd: rawnand: pl353: Removed unused variable

commit a654f35418b7283ce37aab24d6296a3dc5944752 from
https://github.com/Xilinx/linux-xlnx.git

Removed unused variable.

Addresses-Coverity:"set_but_not_used"
Signed-off-by: Amit Kumar Mahapatra <amit.kumar-mahapatra@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/mtd/nand/raw/pl353_nand.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/mtd/nand/raw/pl353_nand.c b/drivers/mtd/nand/raw/pl353_nand.c
index 4bb12a508fe1..936cb78e35ce 100644
--- a/drivers/mtd/nand/raw/pl353_nand.c
+++ b/drivers/mtd/nand/raw/pl353_nand.c
@@ -788,7 +788,6 @@ static int pl353_nand_exec_op_cmd(struct nand_chip *chip,
 	int i;
 	u32 col = 0, row = 0;
 	u32 naddrs = 0;
-	u8 val;
 
 	memset(&nfc_op, 0, sizeof(struct pl353_nfc_op));
 	for (op_id = 0; op_id < subop->ninstrs; op_id++) {
@@ -816,7 +815,6 @@ static int pl353_nand_exec_op_cmd(struct nand_chip *chip,
 			i = nand_subop_get_addr_start_off(subop, op_id);
 			naddrs = nand_subop_get_num_addr_cyc(subop,
 							     op_id);
-			val = instr->ctx.addr.addrs[i];
 			for (i = 0; i < min_t(unsigned int, 4, naddrs); i++)
 				col |= instr->ctx.addr.addrs[i] << (8 * i);
 
-- 
2.31.1

